frontend/test/resolution/testCatch.cpp

```diff
@@ -376,7 +375,6 @@ static void test11(Parser* parser) {
                           }
                         }
               )"""";
-  ctx->advanceToNextRevision(false);
```

Comment:
Can we thread through and re-use a Context to be able to use `advanceToNextRevision` and thus save ourselves some work?

Reply:
``buildStdContext`` is [itself updated](https://github.com/chapel-lang/chapel/pull/26269/files#diff-6843a0340d0ef47d59be4de1a57ed5e37804f307dd540c844ef93a6759aa56de) to invoke ``advanceToNextRevision`` and reload the standard modules.

Reply:
In my experience thus far, it should only add a couple of seconds to a parallel ``test-frontend``. Serially it went from ~65s to ~100s on my local machine.

Reply:
Oh, beautiful. I missed that.